run.ubuntu.sh

```diff
@@ -8,24 +38,24 @@ fi
 
 help_message="Usage: $0 [--run] [--install]"
 
-mode="run"
-if [ $# -ge 2 ]; then
-    echo "Error: $0 takes at most 1 argument."
-    echo "$help_message"
-    exit
-elif [ $# -eq 1 ]; then
-    if [ "$1" = "--run" ]; then
-        mode="run"
-    elif [ "$1" = "--install" ]; then
-        mode="install"
-    elif [ "$1" = "--debug" ]; then
-        mode="debug"
-    else
-        echo "Error: Invalid argument '$1'!"
-        echo "$help_message"
-        exit
-    fi
-fi
+
```

Comment:
The functionality is still available, with some syntax modifications. Since I used "getopts" to parse flags passed for the VM IP Address and Admin token, we thought it would be a good idea to switch to getopts completely. So now, "--install" can be replaced by "-install" and "--debug" can be replaced by "-debug". There is a small catch here which I'll tell you about in our meeting